docs/configuration/index.md

```diff
@@ -1152,6 +1152,7 @@ There are additional configs for autoscaling (if it is enabled):
 |`druid.supervisor.taskUnhealthinessThreshold`|The number of consecutive task failures before the supervisor is considered unhealthy.|3|
 |`druid.supervisor.storeStackTrace`|Whether full stack traces of supervisor exceptions should be stored and returned by the supervisor `/status` endpoint.|false|
 |`druid.supervisor.maxStoredExceptionEvents`|The maximum number of exception events that can be returned through the supervisor `/status` endpoint.|`max(healthinessThreshold, unhealthinessThreshold)`|
+|`druid.supervisor.enableIdleBehaviour`|Whether supervisor should be turned idle if stream is idle for configured time.|false|
```

Comment:
Can this property be set independently per supervisor?

Reply:
This is per cluster, It can't be individually enabled or disabled per supervisor after the design shift, I think we should another config in supervisor spec for the same. Thanks for pointing it out.